stdlib/public/core/PrefixWhile.swift

```diff
@@ -149,6 +149,10 @@ extension LazyPrefixWhileCollection {
   }
 }
 
+extension LazyPrefixWhileSequence where Base: Collection {
+  public typealias Indices = DefaultIndices<LazyPrefixWhileSequence<Base>>
```

Comment:
Same (non-essential) question as above.

Reply:
Unfortunately, these extensions seem to have unique bounds.